samples/dserver.d

```diff
@@ -116,52 +116,40 @@ public:
 ULONG g_cObj =0;
 ULONG g_cLock=0;
 
+import core.sys.windows.dll;
 HINSTANCE g_hInst;
 
-extern (C)
-{
-void *_atopsp;
-void gc_init();
-void gc_term();
-void _minit();
-void _moduleCtor();
-void _moduleUnitTests();
-}
-
-extern (Windows) :
+extern (Windows):
 
 BOOL DllMain(HINSTANCE hInstance, ULONG ulReason, LPVOID pvReserved)
 {
-    // _atopsp = (void*)&hInstance;
-
     switch (ulReason)
     {
         case DLL_PROCESS_ATTACH:
-            gc_init();
-            _minit();
-            _moduleCtor();
-
-            //      _moduleUnitTests();
+            g_hInst = hInstance;
+            dll_process_attach( hInstance, true );
```

Comment:
No proxy (correctly) involved here, and dll_process_attach never meant to work with it.